src/stories/Toast/Toast.style.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import theme from "../theme";
+
+export const ToastStyle = styled.button`
```

Comment:
@PRABgit01 add Styled add in prefix instead of suffix

Reply:
ok